dist/js/ui-components.js

```diff
@@ -0,0 +1,3810 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
You need to remove this file from git, and rebase so that you get the `.gitiignore` entry for it ;).

We can't merge with this here..